.eslintrc.js

```diff
@@ -172,6 +184,10 @@ module.exports = {
     '@typescript-eslint/no-empty-interface': 'off',
     '@typescript-eslint/no-namespace': 'off',
     '@typescript-eslint/ordered-imports': 'off',
+    '@typescript-eslint/no-shadow': [
+      'error',
+      { ignoreTypeValueShadow: true, ignoreFunctionTypeParameterNameValueShadow: true },
+    ],
```

Comment:
Why?

Reply:
Cause you love shadows?!?! I would have just made it a straight error, but ... you love shadows :)

Reply:
I was trying to find the equivalent rule to our old rule about shadows being ok when they don't change the outer scope, but this was the closest I could get with the @typescript-eslint module